guides/puppet-web-apps/html-views/README.md

```diff
@@ -0,0 +1,98 @@
+# HTML Views
+
+This page gives an explaination of partial HTML views and view-models, or, in other words, nested pages.
+
+## Partial HTML View
```

Comment:
:neckbeard: Removing the special meaning of the word partial. 

Could this section be renamed from "Partial HTML view" to "HTML View Definition"?

Also, the next sentence could begin with "A HTML View Definition is a valid HTML document..."

----

At the end of this section we can add a paragraph:

> A "partial" HTML view is a view that has been nested as part of a bigger view. See **Adding the Blending Point**